epyqlib/pm/parametermodel.py

```diff
@@ -325,6 +332,13 @@ class Group(epyqlib.treenode.TreeNode):
             ),
         ),
     )
+    comment = attr.ib(
```

Comment:
Let's change this to be `manual_description` instead of `comment`, which is how it will be used and it will match up better with the other two classes (`Parameter`, `TableArrayElement`).

Reply:
https://github.com/epcpower/stlib/pull/46/commits/a741ac829cb0fd3c0e500891c2e1db88ef077db5